### PR TITLE
models: fix JSON fields

### PR DESCRIPTION
**Pull request by hachreak**
Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: .travis.install-mysql-5.7.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```
Comment (jirikuncar) [inline .travis.install-mysql-5.7.sh]: - missing license
- `sh` should be enough
- `chmod +x`

File: .travis.yml [hunk 2]
```diff
@@ -56,6 +56,7 @@ python:
   - "3.5"
 
 before_install:
+  - bash .travis.install-mysql-5.7.sh
```
Comment (jirikuncar) [inline .travis.yml]: `./.travis.install-mysql-5.7.sh`

**Comment by hachreak** [comment]
ping @jirikuncar

**Comment by jirikuncar** [comment]
@hachreak can you verify the column type on PostgreSQL with before/after this patch and provide the Alembic recipe?

File: .travis.yml [hunk 3]
```diff
@@ -56,7 +56,7 @@ python:
   - "3.5"
 
 before_install:
-  - ./.travis.install-db-5.7.sh
+  - sudo ./.travis.install-db-5.7.sh
```
Comment (jirikuncar) [inline .travis.yml]: I don't think you want to run everything as sudo. Especially the `wget`.

**Comment by hachreak** [comment]
@jirikuncar I wrote the Alembic recipe, but I have an issue with mysql 5.7 installation.

`Access denied for user 'travis'@'%' to database 'invenio'`

**Comment by egabancho** [comment]
https://github.com/inveniosoftware/invenio-webhooks/pull/53

**Pull request closed without merging**